### ISSUE TITLE
Problems with "Ladies & Crooners" frech jazz webradio

### ISSUE THREAD

**Issue by dagisr** [Edzelf/Esp-radio#161]
This works with VLAN in my laptop:
http://icepe4.infomaniak.ch:80/jazzladiescrooners-high

but not with esp-radio.

Please help

**Comment by perlix** [comment]
Looks like your link is being forwarded. 
Try http://jazzladiescrooners.ice.infomaniak.ch/jazzladiescrooners-high.mp3

**Comment by Edzelf** [comment]
Right!  jazzladiescrooners.ice.infomaniak.ch/jazzladiescrooners-high.mp3 works.

**Comment by dagisr** [comment]
YES! it did

Thank you for very fast help
